tests/pact/tc.repl

```diff
@@ -335,3 +335,36 @@
  "rtc object lists"
  1
  (rtc-object-list (let ((a {'a:1}) (b {'a: 2})) [a b])))
+
+;; A module with incorrect annotations for a function return type.
+;; Runtime typechecking for function return types was added in
+;; pact-4.7.
+(module tc-test-function-return-types gov
+  (defcap gov () true)
+  (defun f:string (a: string) (+ 1 2))
+  (defun g:integer (a: integer) a))
+
+;; At pact version 4.6 and earlier, the functions with incompatible type
+;; annotations should compose.
+(env-exec-config ["DisablePact45"])
+(expect
+  "composing type-incompatible, body-incompatible functions succeeds evaluation"
+  3
+  (tc-test-function-return-types.g (tc-test-function-return-types.f "")))
+
+(expect
+  "`compose` on type-incompatible, body-incompatible functions succeeds evaluation"
+  3
+  (compose (tc-test-function-return-types.f) (tc-test-function-return-types.g) ""))
+
+;; After pact version 4.6 and earlier, the functions with incompatible type
+;; annotations should no longer compose.
+(env-exec-config ["DisablePact46"])
```

Comment:
and here you would have no flags at all `(env-exec-config [])` which gives the "latest functionality".